src/Random.elm

```diff
@@ -522,13 +525,45 @@ you use a seed, you'll get another one back.
 initialSeed : Int -> Seed
 initialSeed x =
     let
-        (Seed state1) =
-            next (Seed 0)
+        -- the default increment magic constant is taken from Numerical Recipes
+        (Seed state1 incr) =
+            next (Seed 0 1013904223)
 
         state2 =
             Bitwise.shiftRightZfBy 0 (state1 + x)
     in
-        next (Seed state2)
+        next (Seed state2 incr)
+
+
+{-| A generator that produces a seed that is independent of any other seed in
+the program. These seeds will generate their own unqiue sequences of random
+values. They are useful when you need an unknown amount of randomness *later*
+but can request only a fixed amount of randomness *now*.
+
+The independent seeds are extremely likely to be distinct for all practical
+purposes. However, it is not proven that there are no pathological cases.
+-}
+independentSeed : Generator Seed
+independentSeed =
+    Generator <|
+        \seed0 ->
+            let
+                gen =
+                    int 0 0xFFFFFFFF
```

Comment:
As far as I know `int` uses two's complement signed integers. Did you mean for the range to go from `0` down to `minInt`? Otherwise, if you meant to generate any 32-bit integer, how about `int minInt maxInt`?